sdk/cosmos/azure-cosmos/azure/cosmos/aio/cosmos_client.py

```diff
@@ -134,14 +142,18 @@ async def __aenter__(self):
     async def __aexit__(self, *args):
         return await self.client_connection.pipeline_client.__aexit__(*args)
 
-    async def close(self):
-        # type: () -> None
+    async def close(self) -> None:
         """Close this instance of CosmosClient."""
         await self.__aexit__()
 
     @classmethod
-    def from_connection_string(cls, conn_str, credential=None, consistency_level=None, **kwargs):
-        # type: (str, Optional[Union[str, Dict[str, str]]], Optional[str], Any) -> CosmosClient
+    def from_connection_string(
+            cls: ClassType,
+            conn_str: str,
+            credential: Union[str, Dict[str, Any]] = None,
```

Comment:
This should also be wrapped in `Optional[...]` since per the type comment, `None` is not a valid value.